src/objects/core/zcl_abapgit_filename_logic.clas.abap

```diff
@@ -61,6 +61,9 @@ CLASS zcl_abapgit_filename_logic IMPLEMENTATION.
     REPLACE ALL OCCURRENCES OF '#' IN lv_name WITH '/'.
     REPLACE ALL OCCURRENCES OF '#' IN lv_type WITH '/'.
     REPLACE ALL OCCURRENCES OF '#' IN lv_ext WITH '/'.
+    " Assume AFF namespace convention
+    REPLACE ALL OCCURRENCES OF '(' IN lv_name WITH '/'.
+    REPLACE ALL OCCURRENCES OF ')' IN lv_name WITH '/'.
```

Comment:
This won't work since ( and ) can be part of the object name. For example: `ztest(name).w3mi.data.json` (see https://github.com/abapGit-tests/W3MI_names).

best to also check the registry with `lv_type`.

Reply:
@mbtools  Would you mind to double-check if UT assert is alright? With the current code I get, however this seam not related to changes in this pr.
<img width="831" alt="image" src="https://user-images.githubusercontent.com/38354196/194568512-62939da7-4fed-4570-8cf8-113ea825af09.png">

Reply:
my fault. should be 

```abap
   cl_abap_unit_assert=>assert_equals(
      exp = 'ztest(name).w3mi.json'
      act = lv_filename ).
```

Reply:
thanks